javascript2-week2/quizApp2.css

```diff
@@ -0,0 +1,78 @@
+body {
+    font-family: 'Arial', sans-serif;
+    background-color: #f4f4f4;
+    padding: 20px;
+    color: #333;
+}
+
+.quiz-form,
+.quiz-questions {
+    background: white;
+    padding: 20px;
+    border-radius: 10px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    margin-bottom: 20px;
+}
+
+.form-group {
+    margin-bottom: 15px;
+}
+
+label {
+    display: block;
+    margin-bottom: 5px;
+}
+
+input[type="text"],
+select {
+    width: 100%;
+    padding: 10px;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    box-sizing: border-box;
+    margin-bottom: 10px;
+}
+
+button {
+    background-color: #5cb85c;
+    color: white;
+    border: none;
+    padding: 10px 20px;
+    border-radius: 4px;
+    cursor: pointer;
+    margin-right: 10px;
+}
+
+button:hover {
+    background-color: #4cae4c;
+}
+
+.option-correct {
+    background-color: #dff0d8;
+}
+
+.option-wrong {
+    background-color: #f2dede;
+}
+
+.question-item {
+    background: #fff;
+    padding: 10px;
+    border-radius: 4px;
+    margin-bottom: 10px;
+    box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+}
+
+.answer-reveal {
+    margin-left: 10px;
+    color: #337ab7;
```

Comment:
The blue on green is not the most fortunate combination of colors. It is quite hard to read what the button says. 
In general, when it comes to colors, you can think about colors and theis meanings. For example a green button suggest a positive action - adding question. Red color would suggest something negative - feedback after selecting wrong anwser. Neutral color that fits the design, for example grey wou;ls suggest a neutral action - like randomizing questions. You can try to incorporate that in your app.